apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerTransaction.cs

```diff
@@ -330,6 +337,10 @@ public Task<DateTime> CommitAsync(CancellationToken cancellationToken = default)
                 {
                     throw new SpannerException(ErrorCode.Internal, "Commit succeeded, but returned a response with no commit timestamp");
```

Comment:
nit: Still you might have some commit stats that you can log? Not sure though.

Reply:
This situation is something that should never happen and would be an indication of an internal (backend) error, so I don't think that is something that we need to take into account.